spec/helpers/moments_helper_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe MomentsHelper do
+  let(:user) { FactoryBot.create(:user) }
+  let(:moment) { FactoryBot.create(:moment, :with_secret_share, user: user) }
```

Comment:
do these need to be persisted? can we use `build` instead and stub the DB calls?